RELEASE.txt

```diff
@@ -28,19 +28,20 @@ the appropriate point in main and the following instructions are still apt.
 
   1. Review and cleanup ``doc/release/release_dev.rst``.
 
-  2. Make a list of merges, contributors, and reviewers by running
-     ``tools/generate_release_notes.py -h`` and following that file's usage.
+  2. Make a list of merges, contributors, and reviewers::
 
-  3. Paste this list at the end of the ``release_dev.txt``.
+       # Set release variables
+       export VERSION=<about-to-be-released version number>
+       export PREVIOUS=<previous version number>
 
-  4. Scan the PR titles for highlights, deprecations, API changes,
-     and bugfixes, and mention these in the relevant sections of the notes.
-     Try to present the information in an expressive way by mentioning
-     the affected functions, elaborating on the changes and their
-     consequences. If possible, organize semantically close PRs in groups.
+       pip install changelist
+       changelist scikit-image/scikit-image v${PREVIOUS} main --version >> doc/source/release_notes/release_dev.txt
+
+  4. Scan the PR titles for highlights and mention these in the
+     relevant sections of the notes.
```

Comment:
```suggestion
     relevant sections of the notes.
     Ideally, all changed API objects are mentioned by name, 
     e.g. a new parameter or a deprecated function. 
```
What do you think? I think being able to search release notes for names is one of the main uses of notes. I fully hope that at release time, this work has already been done with the pull request summary. At least I intend to monitor relevant labels in that regard.